indico/modules/categories/forms.py

```diff
@@ -52,6 +52,10 @@ class CategorySettingsForm(IndicoForm):
                                         description=_('Allow users to view all the events descending from this '
                                                       'category in one single page. This is not recommended on large '
                                                       'categories with thousands of events.'))
+    show_future_months = IntegerField(_('Future months threshold'), [NumberRange(min=0)],
+                                      description=_('Future months to show by default on the category page. If no '
+                                                    'events are found, the next 1 month after with events will be '
+                                                    'shown.'))
```

Comment:
Not sure if this is really clear. I mean I understand what it does after looking at the code, but as a user I don't think I'd have an idea what this setting does and if/why I should change it...

Reply:
Yeah, the 1 month after also makes it a little less straightforward. How about turning it around?

> Events past the threshold will be hidden by default to avoid clutter, the user can click to expand them. If no events are found within this threshold, it is extended to show the first month with events.

Reply:
sounds good!